Resources/Prototypes/Access/medical.yml

```diff
@@ -9,7 +9,7 @@
 - type: accessLevel
   id: Chemistry
   name: id-card-access-level-chemistry
-  
+
```

Comment:
consider un-touching this whitespace

Reply:
There's nothing there. I'm not even sure how to undo that.